src/libdoc/phsprt.gls17

```diff
@@ -1,235 +0,0 @@
-
```

Comment:
Why is this file deleted?

Reply:
Otherwise, this looks great and I approve it.

Reply:
PHSPRT GLS17 and PHSPRT GLS18 were both in the repository - GLS17 doesn't seem to be used explicitly, and GLS18 is updated in a later commit. The difference between the two is as follows:

```
--- src/libdoc/phsprt.gls17	2018-07-27 22:51:03.968255753 +0100
+++ src/libdoc/phsprt.gls18	2018-07-27 22:51:03.968255753 +0100
@@ -24,7 +24,9 @@
 ;;; It takes zero to two arguments, one being the sun position
 ;;; (the output of SUN in BKPH's SUN package), and the other a file.
 
-(declare (*lexpr timestamp datimprinc phaseprinc sunposprinc) (newio t))
+(declare (*lexpr timestamp datimprinc phaseprinc sunposprinc)
+ ;;;(newio t)
+)
 
 (defprop moonphase (phase fasl dsk liblsp) autoload)
 (defprop sun-now-here (sun fasl dsk liblsp) autoload)
```

Reply:
Ah, right.  I had kept the old version there temporarily, but forgot to delete it. Thanks.